05-Jason-Luo/wk05/2-tue/mountains/app/views/mountains/edit.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1>Edit mountain</h1>
+
+<form action="<%= mountain_path (@mountain.id) %>" method="post">
+    <label>
+        Name:
+        <input name="name" required value ="<%= @mountain.name %>">
+    </label>
+
+    <label>
+        Image:
+        <input type="url" name="image" value ="<%= @mountain.image %>"> 
+    </label>
+
+
+    <label>
+        Height:
+        <input type="number" name="height" value ="<%= @mountain.height %>">
+    </label>
+
+    <label>
+        Lowest Temperature:
+        <input type="number" name="lowestTemp" value ="<%= @mountain.lowestTemp %>">
+    </label>
+
+    <label>
+        Location:
+        <input name="location" value ="<%= @mountain.location %>">
+    </label>
+
+    <label>
+        Mountain Range:
+        <input name="range" value ="<%= @mountain.range %>">
+    </label>
+
+    <button>Update mountain</button>
+</form>
```

Comment:
You can put this is a form partial so that you don't need to repeat the form across both your edit and new mountain files.
See [Joel's example](https://github.com/wofockham/sei-57/blob/main/08-rails/moma/app/views/works/_form.html.erb) from class.